docs/guidance/assess-against-ids.md

```diff
@@ -1,9 +1,78 @@
+
+### (1) Check your data formats
+
+First, check that the disclosures you want to analyse are in the correct format. If they are not in the correct format, you will need to convert the data.
+
+#### Project level data
+
+Check whether the project level data is published using the [project level data specification](../projects/index.md)
+
+```eval_rst
+.. admonition:: Tip
+    :class: Tip
+
+    .. markdown::
+
+      You can use a [JSON schema validator](https://json-schema.org/implementations.html#validators) to check that whether your data is published in the correct format.
+
+```
+
+If the data isn’t published using the project level data specification, use the [mapping template](https://docs.google.com/spreadsheets/d/1xHLf_w193pp97zfzhLc_LI-yEXrR_eyscga06Qo1blk/copy) to map the data to the specification and create an OC for Infrastructure JSON file for each project.
+
+<!-- TODO: replace link to google sheets with link to OCP website resources section -->
+
+#### Contracting data
+
+Check whether the contracting data is published using OCDS (Tip: You can use the OCDS data review tool for this).
+
+```eval_rst
+.. admonition:: Tip
+    :class: Tip
+
+    .. markdown::
+
+      You can use the [OCDS Data Review Tool](http://standard.open-contracting.org/review/) to check that whether your data is published in OCDS format.
+
+```
+
+If the contracting data is published using OCDS then use it to populate the contracting processes section of the project level data, following the guidance on [using contracting data to understand infrastructure projects](contracts-to-projects.md).
+
+<!-- TODO: update link once input mapping added to documentation -->
+
+If the data isn’t published using OCDS, use the [mapping template](https://docs.google.com/spreadsheets/d/1xHLf_w193pp97zfzhLc_LI-yEXrR_eyscga06Qo1blk/copy) to map the data to the [contracting processes section](../projects/reference.md#contractingprocess) of the project level data specification and add the data to the OC for Infrastructure JSON files for each project.
+
+<!-- TODO: replace link to google sheets with link to OCP website resources section -->
+
+### (2) Check which elements of IDS are disclosed
+
+Use the [OC for Infrastructure - IDS mapping](../cost/index.md) to construct queries to determine which elements of the IDS are provided in the data.
+
+For example, the CoST IDS mapping describes how the project name element of the IDS should be disclosed:
+
+> Project Level: Publish as `title`
+
+Based on this description, the following pseudo code checks a folder containing OC for Infrastructure JSON files to count the number of  projects in which the project name is disclosed:
 
-```
\ No newline at end of file
+```
+for each json file in folder
+    load json
+if top-level "title" field exists in json and its value is not an empty string
```

Comment:
`if` should be indented.

Reply:
Addressed in https://github.com/open-contracting/infrastructure/pull/46/commits/5c760acb02f31f25ee19ea29362dc314da9f633c